x/twap/logic.go

```diff
@@ -53,7 +53,12 @@ func getSpotPrices(ctx sdk.Context, k types.AmmInterface, poolId uint64, denom0,
 			sp1 = sdk.ZeroDec()
 		}
 	}
-	// if sp0.GT(gammtypes.MaxSpotPrice)
+	if sp0.GT(types.MaxSpotPrice) {
+		sp0 = types.MaxSpotPrice
```

Comment:
If we go with the assumption of a possible different spot price backend, it might not return an error when reaching this case. So should we then set `latestErrTime` to the current block time here?

Reply:
Good catch